migration/v0.7.x-v0.8.x/migrate-apps.md

```diff
@@ -0,0 +1,110 @@
+   - Remove `prometheus.retention.alertManager` from `wc-config.yaml`.
+   - Update `falco.alerts.hostPort` to `"http://kube-prometheus-stack-alertmanager.monitoring:9093"` in `wc-config.yaml`
+   - Please update the InfluxDB values, and make sure any new values match your old values when applicable.
+     - `influxDB.address` removed
+     - `influxDB.metrics.sizeWc` replaced by `influxDB.retention.sizeWC`
+     - `influxDB.metrics.sizeSc` replaced by `influxDB.retention.sizeSC`
+     - `influxDB.retention.ageWc` replaced by `influxDB.retention.durationWC`
+     - `influxDB.retention.ageSc` replaced by `influxDB.retention.durationSC`
+   - The config for opendistro has been changed.
+     Please update the new defaults to match your old values.
+     - `elasticsearch.tolerations` removed
+     - `elasticsearch.nodeSelector` removed
+     - `elasticsearch.affinity` removed
+     - `elasticsearch.storageClass` replaced by `elasticsearch.dataNode.storageClass`
+     - `elasticsearch.retention.kubeAuditSize` replaced by `elasticsearch.curator.retention.kubeAuditSizeGB`
+     - `elasticsearch.retention.kubeAuditAge` replaced by `elasticsearch.curator.retention.kubeAuditAgeDays`
+     - `elasticsearch.retention.kubernetesSize` replaced by `elasticsearch.curator.retention.kubernetesSizeGB`
+     - `elasticsearch.retention.kubernetesAge` replaced by `elasticsearch.curator.retention.kubernetesAgeDays`
+     - `elasticsearch.retention.otherSize` replaced by `elasticsearch.curator.retention.otherSizeGB`
+     - `elasticsearch.retention.otherAge` replaced by `elasticsearch.curator.retention.otherAgeDays`
+8. Upgrade Workload cluster applications
+   ```bash
+   # Delete old prometheus-operator release and related resources
+   ./bin/ck8s ops helm wc uninstall prometheus-operator -n monitoring
+   ./bin/ck8s ops kubectl wc delete service/prometheus-operator-kubelet -n kube-system
+   ./bin/ck8s ops kubectl wc delete pvc/prometheus-prometheus-operator-prometheus-db-prometheus-prometheus-operator-prometheus-0 -n monitoring
+
+   # Delete old nginx release and namespace
+   ./bin/ck8s ops helm wc -n nginx-ingress uninstall nginx-ingress && ./bin/ck8s ops kubectl wc delete namespace nginx-ingress
+
+   # Bootstrap to get new CRDs and new namespace
+   ./bin/ck8s bootstrap wc
+
+   # Install new release of nginx
+   ./bin/ck8s ops helmfile wc -l app=ingress-nginx -i apply
+
+   # Install new release of prometheus
+   ./bin/ck8s ops helmfile wc -l app=kube-prometheus-stack -i apply
+
+   # Update everything else
+   ./bin/ck8s apply wc
+   ```
+9. Upgrade Service cluster applications
+   ```bash
+   # Delete old prometheus-operator release and related resources
+   ./bin/ck8s ops helm sc uninstall prometheus-operator -n monitoring
+   ./bin/ck8s ops kubectl sc delete service/prometheus-operator-kubelet -n kube-system
+   ./bin/ck8s ops kubectl sc delete pvc/prometheus-prometheus-operator-prometheus-db-prometheus-prometheus-operator-prometheus-0 -n monitoring
```

Comment:
An awsomely long name for a pvc :rofl:

Reply:
Indeed :rofl: no less than 5 `prometheus`!

Reply:
Actually its **6** :laughing: